src/frontend/packages/lib_video/src/components/common/VideoWidgetProvider/LocalizedTimedTextTrackUpload/index.tsx

```diff
@@ -112,6 +113,13 @@ export const LocalizedTimedTextTrackUpload = ({
             timedTextTrackId,
             event.target.files[0],
             video.id,
+            (presignedPost) => {
+              timedTextTrackUploadEnded(
+                video.id,
+                timedTextTrackId as string,
```

Comment:
What happen if `timedTextTrackId` is undefined ? Should we add a condition?

Reply:
This is a typescript problem, if u look at the code above their is already a condition that check if the timedTextTrackId is undefined (if yes then we create the resource to not be undefined). But for some reasons the type is still 'string or undefined' inside the lamda